migrator/src/main/java/com/google/phonenumbers/migrator/CommandLineMain.java

```diff
@@ -56,6 +59,26 @@
       description = "The BCP-47 country code the given phone number(s) belong to (e.g. 44)")
   String countryCode;
 
+  @ArgGroup()
+  OptionalParameterType optionalParameter;
+
+  static class OptionalParameterType {
+    @Option(names = {"-l", "--lenientExport"},
+        description = "boolean flag specifying that text files created after the migration process"
+            + " for standard recipe --file migrations should contain the migrated version of a given"
+            + " phone number, regardless of whether the migration resulted in an invalid phone number."
+            + " By default, a strict approach is used and when a migration is seen as invalid, the"
+            + " original phone number is written to file. Invalid numbers will be printed at the"
```

Comment:
I think tweaking this just a bit to be more clear, that all migrated numbers will be in their original order while migrated invalid numbers will be added to to bottom of the text file.

I also think we should double check with David on this, maybe you can ask him the next time you talk to him if this seems reasonable depending on the different use cases.